eclipse/sw360
# fix(spdx): import large spdx rdf files

Signed-off-by: Kouki Hama <kouki1.hama@toshiba.co.jp>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: 　https://github.com/eclipse/sw360/issues/1011
spdx/tools takes a lot of time and resources to parse big rdf file (ie. >15M).
Use javax.xml.parsers and org.w3c.dom.Document for faster rdf reading,
and implement parser to get only information we need (licenses and copyrights)

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer?

Import spdx files (ex,  made by fossology) from release  attachment :

Component/release -> Clearing details -> click show license info -> get results 

Generating the catalogue ("Generate License Info") with spdx rdf file


> Have you implemented any additional tests?

Import large spdx file from release attachment :
 
Component/release -> Clearing details -> click show license info -> get results

Generating the catalogue ("Generate License Info") with large spdx rdf file

### Checklist
Must:

### Thanks 
@dothanhtrung  and @tienlee  contributed and helped a lot.   

### Note 
We would like to explain about  this pull request on Feb 3rd sw360 telco by using https://github.com/eclipse/sw360/files/5877677/Meeting_SPDXParser_20210127.pdf
